hs_app_timeseries/models.py

```diff
@@ -971,6 +971,7 @@ def series_names(self):
         # uploaded csv file. This list becomes an empty list
         # once metadata changes are written to the blank sqlite file as
         # part of the sync operation.
+        self.refresh_from_db()
```

Comment:
I don't think we should have a refresh_from_db in here.

Reply:
See below.

Reply:
As my comment above